p/api/greader.php

```diff
@@ -216,9 +216,13 @@ function token($conf) {
 function checkToken($conf, $token) {
 //http://code.google.com/p/google-reader-api/wiki/ActionToken
 	$user = Minz_Session::param('currentUser', '_');
+	if ($user !== '_' && $token == '') {
```

Comment:
== or ===?

Reply:
Strict for the first comparison, loose for the second (match empty string and 0 in particular)